.travis.yml

```
@@ -31,7 +31,7 @@ before_install:
   - geckodriver --version
   # Install chromedriver from http://chromedriver.chromium.org/downloads
   - mkdir -p tmp/chromedriver
-  - curl -f -L -o chromedriver_linux64.zip https://chromedriver.storage.googleapis.com/76.0.3809.68/chromedriver_linux64.zip
+  - curl -f -L -o chromedriver_linux64.zip https://chromedriver.storage.googleapis.com/77.0.3865.10/chromedriver_linux64.zip
```

Latest version is `77.0.3865.40`

```suggestion
  - curl -f -L -o chromedriver_linux64.zip https://chromedriver.storage.googleapis.com/77.0.3865.40/chromedriver_linux64.zip
```